src/transformers/models/align/processing_align.py

```diff
@@ -16,8 +16,32 @@
 Image/Text processor class for ALIGN
 """
 
-from ...processing_utils import ProcessorMixin
-from ...tokenization_utils_base import BatchEncoding
+from typing import List, Union
+
+
+try:
+    from typing import Unpack
+except ImportError:
+    from typing_extensions import Unpack
+
+from ...image_utils import ImageInput
+from ...processing_utils import (
+    CommonKwargs,
+    ImagesKwargs,
+    ProcessingKwargs,
+    ProcessorMixin,
+    TextKwargs,
+)
+from ...tokenization_utils_base import BatchEncoding, PreTokenizedInput, TextInput
+
+
+class AlignProcessorKwargs(ProcessingKwargs, CommonKwargs, TextKwargs, ImagesKwargs, total=False):
```

Comment:
last nit, but ProcessingKwargs inherits from CommonKwargs and TextKwargs and ImageKwargs. We only need processing kwerag no?

Reply:
yes, that should work now! checking

Reply:
yes, works :D